server/src/localization.cpp

```diff
@@ -3,17 +3,94 @@
 //
 
 #include "localization.hpp"
+#include "data_store.hpp"
+
+extern insNode_t * insNoderoot;
 
 namespace ins_service
 {
 
+wifiParams_t * Localization::FillNodeDataPoints(wifiParams_t *  wifiNodeBlock, AccessPointRssiListPair mac_rssi_)
```

Comment:
I don't feel comfortable at all playing around with raw pointers in C++. I don't think they should be used at all.

Reply:
Why?